WooCommerce/Classes/Analytics/WooAnalyticsStat.swift

```diff
@@ -121,6 +121,7 @@ public enum WooAnalyticsStat: String {
     case settingsSelectedStoreTapped            = "settings_selected_site_tapped"
     case settingsContactSupportTapped           = "main_menu_contact_support_tapped"
     case settingsCardReadersTapped              = "settings_card_readers_tapped"
+    case settingsCouponManagementTapped         = "settings_coupon_management_tapped"
```

Comment:
I think that you can remove this type of tracking. Usually, we add the tracking at the end of a project and we discuss it also with Android developers. So feel free to ignore everything regarding the tracking. 👍